src/detectors/IScanningStrategy.ts

```diff
@@ -0,0 +1,13 @@
+export enum ServiceType {
```

Comment:
I would suggest changing the name of this file from `IScanningStrategy.ts` to `ScanningStrategy.enum.ts`. The `I` has implied an interface class file, whereas the file contains only enumerations. But conventionally I am not sure, maybe @prokopsimek will have a different take on it.

Reply:
It's true that the `I` prefixed files should define a class interface but it isn't crucial.

Reply:
I know it's not the best name but I think we already use this "pattern" also with other names of files. I think there could also be more interfaces or enum in this file, so if you don't mind, I'll let it as is. @ryzzaki